src/EditorFeatures/CSharpTest/SignatureHelp/WithElementSignatureHelpProviderTests.cs

```diff
@@ -39,12 +39,42 @@ void Goo()
             </Workspace>
             """;
 
-        await TestAsync(markup, [new("List<int>(int capacity)", string.Empty, null, currentParameterIndex: 0)]);
+        await TestAsync(markup, [
+            new("List<int>()", string.Empty, currentParameterIndex: 0),
```

Comment:
why did the empty list get added?

Reply:
we decided to allow `with()` to bind, not just `with(capacity: ...)` :)